returnn/tf/layers/basic.py

```diff
@@ -5728,6 +5731,24 @@ class LossLayer(LayerBase):
   But this loss will not be used as a loss by the updater.
   If you want to use it as a loss, you can use the :class:`AsIsLoss`,
   i.e. write ``"loss": "as_is"``.
+
+  Note that the loss options for the wrapped loss need to be provided via ``loss_opts_``,
+  and it does not apply any reduce function.
+
+  .. note::
+
+    The ``LossLayer`` might be deprecated in the future in favor of implementing the losses as actual layers.
+
+    If you want to define a loss inside the network, it is recommended to define it explicitly via an ``eval`` layer.
+    An example could be:
+
+    ``"squared_error_loss": {"class": "eval", "eval": "(source(0) - source(1)) ** 2", "from": ["output", "data:classes"]}``
+
+    Followed by an e.g. mean reduce if needed:
+
+    ``"mse_loss": {"class": "reduce", "mode": "mean", "axis": "F", "from": "squared_error_loss"}``
```

Comment:
As I explained, remove this example about the reduction. This is misleading. The user should not do this. This is not the same as the normal loss calculation. (Did you see my example, which shows that this is not the same?)

If you want to write something about reduction, write that the user should not do it, or if he does, it would be different to what `Loss` does.

Reply:
Yes, this is just the code to replicate what the ``MeanSquaredError`` is doing. I do not see how this could be misleading.